HW8_Airport/Air_Simulation/Baggage.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Air_Simulation
+{
+    class Baggage
+    {
+        public int weight;
+        private string[] forbiddenBag = new string[] { "drugs", "alcohol", "gasSpray", "ammunition", "toxicSubstance", "sharpObject" };
```

Comment:
as a private fields - me just use the same approach to declare a field:
private string[] forbiddenBag => `private string[] _forbiddenBag`